.circleci/config.yml

```diff
@@ -19,6 +19,7 @@ publish_release_task: &publish_release_task
   name: Publish Release Artifacts
   command: make publish-release-artifacts
 
+executorType: machine
```

Comment:
Why do we need this change? Can you document if this is required why.

Reply:
Yeap, left link in commit, here it is
https://www.testcontainers.org/supported_docker_environment/continuous_integration/circle_ci/

Reply:
Can we add the link as a comment below this line? Better be safe than sorry (also would prevent future changes from removing it 'accidentally')

Reply:
Yeap i'll add

Reply:
Can you give a try to see if it does not work without this change?

Reply:
i was doing it, it was not working until commit
https://github.com/open-telemetry/opentelemetry-java/pull/1038/commits/68caa9089b7ef16e0c7fee4df6af16cf3868e503
there was some error like docker not running